packages/plugin-core/src/test/suite-integ/OpenLink.test.ts

```diff
@@ -20,7 +20,7 @@ suite("OpenLink", function () {
       onInit: async ({}) => {
         const cmd = new OpenLinkCommand();
         const { error } = await cmd.execute();
-        expect(error!.message).toEqual("nothing selected");
+        expect(error!.message).toEqual("no valid path or URL selected");
```

Comment:
can you add a test for the added functionality here? You can see how we add tests with selection enabled [here](https://github.com/dendronhq/dendron/pull/1317/files). instructions for running specific test [here](https://wiki.dendron.so/notes/cb22bd36-d45a-4ffd-a31e-96c4b39cb7fb.html#run-specific-test)